### PR TITLE
Debugger stops on assertions during test running step-by-step (closes #1288)

### LINKED ISSUE #1288: Debugger should stops on assertions during test running step-by-step
### Are you requesting a feature or reporting a bug?

feature

### What is the current behavior?

assertions are skipped

### What is the expected behavior?

see subject

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin @AndreyBelym

### CODE REVIEW

File: src/test-run/index.js [hunk 1]
```diff
@@ -361,13 +372,12 @@ export default class TestRun extends Session {
...
             return this._enqueueSetDialogHandlerCommand(command, callsite);
 
-        if (command.type === COMMAND_TYPE.assertion) {
-            var executor = new AssertionExecutor(command, callsite);
 
-            executor.once('start-assertion-retries', timeout => this._enqueueCommand(new ShowAssertionRetriesStatusCommand(timeout)));
-            executor.once('end-assertion-retries', success => this._enqueueCommand(new HideAssertionRetriesStatusCommand(success)));
+        if (command.type === COMMAND_TYPE.assertion) {
+            if (this.debugging)
+                await this._enqueueCommand(command, callsite);
```
Comment (AlexanderMoskovkin) [inline src/test-run/index.js]: lgtm overall, but I'd prefer to add a comment here

File: src/test-run/index.js [hunk 2]
```diff
@@ -361,13 +372,14 @@ export default class TestRun extends Session {
...
             return this._enqueueSetDialogHandlerCommand(command, callsite);
 
-        if (command.type === COMMAND_TYPE.assertion) {
-            var executor = new AssertionExecutor(command, callsite);
 
-            executor.once('start-assertion-retries', timeout => this._enqueueCommand(new ShowAssertionRetriesStatusCommand(timeout)));
-            executor.once('end-assertion-retries', success => this._enqueueCommand(new HideAssertionRetriesStatusCommand(success)));
+        if (command.type === COMMAND_TYPE.assertion) {
+            // NOTE: we should send assertion command on client only in case of execution test
+            // step-by-step in debugging mode to show debugging status UI in status panel
```
Comment (VasilyStrelyaev) [inline src/test-run/index.js]:
```
            // NOTE: we should send the assertion command to the client only if the test is executed
            // step-by-step in debugging mode to show debugging status in the status panel
```

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

**Pull request merged**